draft-ietf-dnsop-svcb-https.md

```diff
@@ -517,24 +509,30 @@ for a service, as performed by the client.  SVCB resolution is implemented as fo
 This procedure does not rely on any recursive or authoritative DNS server to
 comply with this specification or have any awareness of SVCB.
 
-Once SVCB resolution has concluded, the client proceeds with connection
-establishment.  Clients SHOULD try higher-priority alternatives first, with
-fallback to lower-priority alternatives.  Clients issue AAAA and/or A
-queries for the selected TargetName, and MAY choose between them using an
-approach such as Happy Eyeballs {{!HappyEyeballsV2=RFC8305}}.
-
 A client is called "SVCB-optional" if it can connect without the use of
 ServiceMode records, and "SVCB-reliant" otherwise.  Clients for pre-existing
 protocols (e.g. HTTPS) SHALL implement SVCB-optional behavior (except as
 noted in {{client-failures}} and {{ech-client-behavior}}).
 
 SVCB-optional clients SHOULD issue in parallel any other DNS queries that might
-be needed for connection establishment.  SVCB-optional clients SHALL append an
-alternative endpoint consisting of the final value of $QNAME, the authority
-endpoint's port number, and no SvcParams, to the list of alternative endpoints, which is
+be needed for connection establishment if the SVCB record is absent.
+
+Once SVCB resolution has concluded, whether successful or not,
+SVCB-optional clients SHALL append to the priority list an
+endpoint consisting of the final value of $QNAME, the authority
+endpoint's port number, and no SvcParams.  (This endpoint will be
 attempted before falling back to non-SVCB connection modes.  This ensures that
 SVCB-optional clients will make use of an AliasMode record whose TargetName has
-A and/or AAAA records but no SVCB records.
+A and/or AAAA records but no SVCB records.)
+
+The client proceeds with connection establishment using the resolved list of
+endpoints.  Clients SHOULD try higher-priority alternatives first, with
+fallback to lower-priority alternatives.  Clients issue AAAA and/or A
+queries for the selected TargetName, and MAY choose between them using an
```

Comment:
Responding to https://github.com/MikeBishop/dns-alt-svc/pull/335#r723694121

It's true that these queries might not be "issued", but I'd rather avoid discussing caches here.  Maybe we can solve this with more precise language:
```suggestion
fallback to lower-priority alternatives.  Clients resolve AAAA and/or A
records for the selected TargetName, and MAY choose between them using an
```